appveyor.yml

```diff
@@ -1,9 +1,9 @@
 environment:
-  MVN_VERSION: 3.3.3
```

Comment:
The apache maven 3.3.3 is missing:
http://apache.mirrors.pair.com/maven/maven-3/
![Uploading 6D68DAA6-A18F-4F24-BD8E-16EFCD743447.png…]()